settings.gradle

```diff
@@ -7,5 +7,6 @@ include 'governator-commons-cli'
 include 'governator-test-junit'
 include 'governator-archaius'
 include 'governator-jetty'
+include 'governator-jersey'
```

Comment:
jersey2 should be different.  I was planning on governator-jersey and governator-jersey2